Could not find artifact net.sf.ehcache:ehcache-terracotta:jar:2.6.2 in central

The dependency ehcache-terracotta:2.6.2 is no longer available in Maven central. Hence, the build using Maven fails (unless you already have the artifact, I assume):

`Downloading: http://repo.maven.apache.org/maven2/net/sf/ehcache/ehcache-terracotta/2.6.2/ehcache-terracotta-2.6.2.pom`
`[WARNING] The POM for net.sf.ehcache:ehcache-terracotta:jar:2.6.2 is missing, no dependency information available`
`Downloading: http://repo.maven.apache.org/maven2/net/sf/ehcache/ehcache-terracotta/2.6.2/ehcache-terracotta-2.6.2.jar`

`[ERROR] Failed to execute goal on project luke: Could not resolve dependencies for project luke:luke:jar:4.10.1: Could not find artifact net.sf.ehcache:ehcache-terracotta:jar:2.6.2 in central (http://repo.maven.apache.org/maven2) -> [Help 1]`

However, after changing the ehcache version in pom.xml to 2.9.0 (most recent, currently), multiple errors of the following type arise:
`src/main/java/org/getopt/luke/plugins/FsDirectory.java:[21,22] package net.sf.ehcache does not exist`

--- a/pom.xml
+++ b/pom.xml
@@ -11,7 +11,7 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <lucene.version>4.10.1</lucene.version>
-        <ehcache.version>2.6.2</ehcache.version>
+        <ehcache.version>2.9.0</ehcache.version>
         <hadoop.version>1.2.1</hadoop.version>
         <log4j.version>1.2.17</log4j.version>
         <rhino.version>1.7R4</rhino.version>
@@ -98,7 +98,6 @@
             <groupId>net.sf.ehcache</groupId>
             <artifactId>ehcache</artifactId>
             <version>${ehcache.version}</version>
-            <type>pom</type>
         </dependency>
 
         <dependency>

